data/lua/wml/modify_unit.lua

```diff
@@ -10,20 +10,20 @@ function wml_actions.modify_unit(cfg)
 
 	local replace_mode = cfg.mode == "replace"
 
-	local function handle_attributes(cfg2, unit_path, toplevel)
-		for current_key, current_value in pairs(wml.shallow_parsed(cfg2)) do
+	local function handle_attributes(handle_attr_cfg, unit_path, toplevel)
+		for current_key, current_value in pairs(wml.shallow_parsed(handle_attr_cfg)) do
 			if type(current_value) ~= "table" and (not toplevel or (current_key ~= "type" and current_key ~= "mode")) then
 				wml.variables[string.format("%s.%s", unit_path, current_key)] = current_value
 			end
 		end
 	end
 
-	local function handle_child(cfg2, unit_path)
+	local function handle_child(handle_child_cfg, unit_path)
```

Comment:
This is a slight improvement, but I don't think including the function name in the argument name is a good idea either.

Reply:
I really don't have any preference on what the name is, so let me know what you'd prefer I change it to.

Reply:
`child_cfg` seems reasonable, for both functions.

Reply:
Alright, done.